.travis.yml

```diff
@@ -26,6 +26,9 @@ install:
     - npm install
     - grunt compile
 
+after_failure:
```

Comment:
I dont think we need this. The error is already clearly visible in the grunt/npm task fails. This just clutters the output.